pkg/monitor/querier.go

```diff
@@ -151,6 +152,11 @@ func (monitor *agentMonitor) GetAgentConditions(ovsConnected bool) []v1beta1.Age
 	}
 }
 
+// GetAgentInfo gets the antrea agent information.
+func (monitor *agentMonitor) GetAgentInfo() *v1beta1.AntreaAgentInfo {
+	return monitor.getAntreaAgentInfo(monitor.GetSelfNode().Name)
+}
```

Comment:
The methods `GetAgentInfo`, `getAntreaAgentInfo`, `getAgentCRD` in monitor would be confusing, maybe you can just use `GetAgentInfo` as name of `getAntreaAgentInfo` method.

Reply:
Yes, I changed them to GetAgentInfo().